packages/modular-scripts/src/build-scripts/buildStandalone.ts

```diff
@@ -199,9 +175,9 @@ export async function buildStandalone(
   // Add dependencies from source and bundled dependencies to target package.json
   const targetPackageJson = (await fs.readJSON(
     path.join(targetDirectory, 'package.json'),
-  )) as CoreProperties;
+  )) as ModularPackageJson;
   // Copy selected fields of package.json over
-  fs.writeJSON(
+  fs.writeJsonSync(
```

Comment:
Why not `await fs.writeJSON`?

Reply:
Is there a benefit to one over the other? 
Not sure why I changed it to that instead of awaiting

Reply:
There is when we enable parallel builds, because async calls can be parallelised by Node, sync calls block the main thread until they return, so they're always serialized.